app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx

```diff
@@ -51,6 +51,65 @@ export class CloudProviderIcon extends React.Component<IServerGroupHeaderProps>
   }
 }
 
+export interface IImageListState {
+  collapsed: boolean;
+}
+
+export class ImageList extends React.Component<IServerGroupHeaderProps, IImageListState> {
+  private toggle() {
+    this.setState((previousState: IImageListState) => {
+      return { collapsed: !previousState.collapsed };
+    });
+  }
+
+  constructor(props: IServerGroupHeaderProps) {
+    super(props);
+    this.state = {
+      collapsed: true,
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  public render() {
+    const { images } = this.props;
+    const { collapsed } = this.state;
+
+    // if we only have 1 image, render as simple string.
+    if (images.length === 1) {
+      return <span>{images[0]}</span>;
+    }
+
+    // otherwise, collapsed by default, with the option to toggle all.
+    return (
+      <>
+        {collapsed && (
+          <>
+            <span>{images.length} containers per pod. </span>
```

Comment:
"container" and "pod" are both container-specific, whereas this header is shared amongst all providers where, for example, "VM Images" or "Instances" might be more apt descriptions.  I suggest slightly tweaking the approach here to render a single image in the collapsed view with a "(+ X more)" link that, when clicked, shows the complete list.  This way we can avoid introducing provider-specific language completely.

Reply:
I see. Yeah, that sounds good!